game/game/gamescript.cpp

```diff
@@ -2590,7 +2597,7 @@ void GameScript::ai_turntonpc(std::shared_ptr<phoenix::c_npc> selfRef, std::shar
   }
 
 void GameScript::ai_outputsvm(std::shared_ptr<phoenix::c_npc> selfRef, std::shared_ptr<phoenix::c_npc> targetRef, std::string_view name) {
-  auto target = findNpc(targetRef);
+  auto target = targetRef==nullptr ? globalOther() : findNpc(targetRef);
```

Comment:
You mention that `NULL` case is specific to G1, so `VersionInfo` need to be used, similar to how `hasZSStateLoop` is implemented